src/app/home/home.component.ts

```diff
@@ -346,6 +392,10 @@ export class HomeComponent implements AfterViewInit {
         }
     }
 
+    /**
+     * Places the chosen stored text in the editor.
```

Comment:
Where is it stored. Maybe in the context where this function is called there is something that is stored somewhere that is being passed as argument, but in the isolated context the function parameter purpose should be a bit more abstract. 
Maybe: i.e.: `"Replaces the text of the editor with the given **writtenText**"`
also add  `"...and reapplies the markings on the new text accordingly" `

Reply:
Can indeed be improved, does `Replaces the text of the editor with the given **writtenText** and generates its markings` does good?

Reply:
sound good*